data/announcements/scheduled.yaml

```diff
@@ -211,7 +211,7 @@ announcements:
 - name: Legacy package repositories shutdown
   startTime: 2023-11-27T00:00:00
   endTime: 2024-02-01T00:00:00
-  style: "background: #d95e00"
+  style: "background: #3371e3"
```

Comment:
This proposed change is incorrect. It alters the color of the announcement banner (click [preview link](https://deploy-preview-44808--kubernetes-io-main-staging.netlify.app/)) instead of the CNFC iframe as requested.